dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -587,11 +596,16 @@ const SwapHomepage = ({
           </a>
           <button
             //disabled={formHasErrors || buyFormHasWarnings || !totalOUSD}
-            className={`btn-blue buy-button mt-2 mt-md-0 w-100`}
+            className={`btn-dark buy-button mt-2 mt-md-0 w-100`}
```

Comment:
was this change intended?

Reply:
From what I can tell the above comment only references the button copy. And the screenshot above is from Uniswap and not our app.